js/bootstrap-datepicker.js

```diff
@@ -1054,7 +1055,7 @@
 					clsName = $.unique(clsName);
 				}
 
-				html.push('<td class="'+clsName.join(' ')+'"' + (tooltip ? ' title="'+tooltip+'"' : '') + ' data-date="' + prevMonth.getTime().toString() + '">' + content + '</td>');
+                html.push('<td class="' + clsName.join(' ') + '"' + (tooltip ? ' title="' + tooltip + '"' : '') + ' data-date="' + prevMonth.getTime().toString() + '" data-title="' + accessibleTitle + '">' + content + '</td > ');
```

Comment:
This builds the the text for a label to be used for a single day in the calendar. e.g. March 16 2017 Thursday -- I mostly followed the recommendations of another datepicker that is accessible